Source/IGListKit/IGListCollectionContext+Refinements.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright (c) Facebook, Inc. and its affiliates.
+// All rights reserved.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree. An additional grant
+// of patent rights can be found in the PATENTS file in the same directory.
+//
+// GitHub:
+// https://github.com/Instagram/IGListKit
+//
+// Documentation:
+// https://instagram.github.io/IGListKit/
+//
+
+import UIKit
+
+extension ListCollectionContext {
+    func dequeueReusableCell<T: UICollectionViewCell>(
+        of cellClass: T.Type, withReuseIdentifier reuseIdentifier: String,
+        for sectionController: ListSectionController, at index: Int) -> T {
+        return self.__dequeueReusableCell(
+            of: cellClass, withReuseIdentifier: reuseIdentifier, for: sectionController, at: index) as! T
```

Comment:
I can imagine that a force cast is not acceptable, but this could just as well be something with a `guard`;

```swift
guard
    let cell = self.__dequeueReusableCell(
        of: cellClass, withReuseIdentifier: reuseIdentifier, 
        for: sectionController, at: index) as? T else {
            fatalError()
}

return cell

Reply:
Yeah, could you update so that it's not a force cast?

Reply:
Done